tensorflow/compiler/tf2tensorrt/convert/op_converter.h

```diff
@@ -25,6 +25,33 @@ limitations under the License.
 #include "tensorflow/compiler/tf2tensorrt/convert/weights.h"
 #include "third_party/tensorrt/NvInfer.h"
 
```

Comment:
There's another PR where these are moved to `tf2tensorrt/common/utils.h`. We should probably prefer that location since other error handeling macros are defined there.

Reply:
Sorry, I am not sure that I understand you correctly... Are you talking about adding these two lines:
```
#include "tensorflow/compiler/tf2tensorrt/convert/weights.h"
#include "third_party/tensorrt/NvInfer.h"
```
to `op_converter.h` (current version of this file does not contain them)
      OR
about moving macros to  `tf2tensorrt/common/utils.h` ?

Reply:
moving macros. the comment is appearing above the code

Reply:
Done